internal/kafka/internal/workers/kafka_mgrs/accepted_kafkas_mgr.go

```diff
@@ -68,6 +68,7 @@ func (k *AcceptedKafkaManager) Reconcile() []error {
 	for _, kafka := range acceptedKafkas {
 		glog.V(10).Infof("accepted kafka id = %s", kafka.ID)
 		metrics.UpdateKafkaRequestsStatusSinceCreatedMetric(constants2.KafkaRequestStatusAccepted, kafka.ID, kafka.ClusterID, time.Since(kafka.CreatedAt))
+		metrics.UpdateKafkaRequestsStatusSinceLastSetMetric(constants2.KafkaRequestStatusAccepted, kafka.ID, kafka.ClusterID)
```

Comment:
For accepted status, this has to be set only in `KafkaService#RegisterKafkaJob()`: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/ee6c732df5c8a67e06a5c2c74e26705e85d2c253/internal/kafka/internal/services/kafka.go#L426